pkg/manifestplugins/markdown/tests/manifests/file.yaml

```diff
@@ -0,0 +1,8 @@
+structure:
+- dir: foo
```

Comment:
Can you add frontmatter and skipValidation to foo directory to test their propagation?